public/js/user.js

```diff
@@ -383,15 +383,15 @@ function saveMarker(e) {
 function updateMarker(id) {
     var json = JSON.stringify({ });
     $.ajax({
-      url: "/marker/"+id,
+      url: "/api/marker/"+id,
       type: "PATCH",
       data: json,
       dataType: "json",
       processData: false,
       contentType: "application/json",
     })
         .done(function(response) {
-          sites[id].siteData = response;
+          sites[id].siteData.date_changed = response.date_changed;
```

Comment:
This isn't related to the issue, but I found that updating a site and then editing it throws an error 'site.auth' not found. Here's a quick fix. :)